src/proof/er/er_proof.cpp

```diff
@@ -220,30 +221,43 @@ void ErProof::outputAsLfsc(std::ostream& os) const
   for (const ErDefinition& def : d_definitions)
   {
     os << "\n    (clausify_rat_elimination_def _ _ _ "
```

Comment:
Where is `clausify_rat_elimination_def` defined?

Reply:
Oh! It isn't. This is definitely broken. I'll find a benchmark that actually triggers this codepath and add it to the test suite.